e2e_test/sink/kafka/create_sink.slt

```diff
@@ -188,6 +188,55 @@ format plain encode json (
   force_append_only='true'
 ) key encode text ;
 
+statement error sink key encode unsupported: JSON, only TEXT and BYTES supported
+create sink sink_bytes_error as (
+  select int8send(id) as id_bytes, * from t_kafka
+) with (
+  connector = 'kafka',
+  properties.bootstrap.server = 'message_queue:29092',
+  topic = 'test-rw-sink-bytes-key-id',
+  primary_key = 'id_bytes')
+format plain encode json (
+  force_append_only='true'
+) key encode json;
+
+statement error
+# The key encode is BYTES, but the primary key has 2 columns. The key encode BYTES requires the primary key to be a single column
+create sink sink_bytes_error as (
+  select int8send(id) as id_bytes, '\x1234'::bytea as other_bytea, * from t_kafka
+) with (
+  connector = 'kafka',
+  properties.bootstrap.server = 'message_queue:29092',
+  topic = 'test-rw-sink-bytes-key-id',
+  primary_key = 'id_bytes, other_bytea')
+format plain encode json (
+  force_append_only='true'
+) key encode json;
```

Comment:
```suggestion
) key encode bytes;
```

Is this why the expected error message is commented out rather than actually checked by sqllogictest?